src/components/SingleVideo.vue

```diff
@@ -48,9 +48,13 @@
 </script>
 
 <template>
-  <div class="videoWrapper">
+  <!--  Note: would like to re-evaluate how this functions on the front-end at some point.
+        I think ideally, we could load in a screenshot or something so YouTube doesn't actually load unless until the user clicks.
```

Comment:
Youtube never loads. Presently we are using a piped instance hosted by us. Mostly because the rss feeds are easier to manage